llvm/lib/Passes/PassBuilderPipelines.cpp

```diff
@@ -1816,6 +1817,17 @@ PassBuilder::buildLTODefaultPipeline(OptimizationLevel Level,
   // in the current module.
   MPM.addPass(CrossDSOCFIPass());
 
+  MPM.addPass(CoroEarlyPass());
+
+  auto Exit = llvm::make_scope_exit([&]() {
```

Comment:
Please call this explicitly where necessary instead of using make_scope_exit.

Reply:
I assume for readability?
@nikic  Or do you know something about make_scope_exit?

I am starting to suspect that my Windows pre-commit errors looks like make_scope_exit didn't execute.

Reply:
make_scope_exit would try to modify the object being returned ... that's probably UB :)